### ISSUE TITLE
Compress COVIDcast API responses

### ISSUE THREAD

**Issue by krivard** [cmu-delphi/delphi-epidata#160]
Especially since county-level responses are super large.

**Comment by sgratzl** [comment]
this is very important in my opinion since it significantly down the COVIDast map. 

see also e.g., https://medium.com/@ali.dev/how-to-setup-caching-gzip-compression-in-apache-web-server-with-htaccess-292a0f689553

https://httpd.apache.org/docs/2.4/mod/mod_brotli.html

in this case https://github.com/cmu-delphi/delphi-epidata/blob/main/src/server/.htaccess would be a start to configure that

**Comment by capnrefsmmat** [comment]
Surprise! delphi.cmu.edu seems to be served by nginx 1.16.1, not Apache, but the map API endpoints are on Apache. I submitted #171 to fix the htaccess for those.

In the nginx config on delphi.cmu.edu we just need to add

```
gzip on;
gzip_types application/json;
```

to the appropriate server block. Pinging @korlaxxalrok. I think this will speed up queries to signals like DV, which can be 200+ kB per day of data. According to a random online checker tool I found, one example day can be compressed to only 14 kB. (Seems implausible, but if random website tools can't be trusted, who can?)
